demos/basics/loading-saving-neo4j.ipynb

```diff
@@ -0,0 +1,2038 @@
+{
```

Comment:
<blockquote>We can use single Cypher query to retrieve an identifier for the source and target of each node is sufficient.</blockquote><p>"We can use a single Cypher query to retrieve the identifiers of the source and target nodes of each edge."?</p><p> </p><blockquote>ensure the DataFrame columns have the easiest name.</blockquote><p>"ensure the DataFrame columns match the defaults"?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/stellargraph/stellargraph/pull/1184/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='13'/>

Reply:
Wow, I really mangled that first sentence!